app/views/recipe_categories/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="column">
+  <h3 class="title has-text-white-ter has-text-right">
+    Recipe categories
+  </h3>
+  <hr class="login-hr">
+  <p class="subtitle has-text-white-ter has-text-right">
+    We already have <%= @recipe_categories.length %> recipe categories!
```

Comment:
I saw in the other PR you left this string as interpolated. 
```erb
<%= "We already have #{@variable}..." %>
```
There is no problem whether is this or the other way, just try to be consistent! :)